kcidb/templates/stable_rc_build.j2

```diff
@@ -0,0 +1,40 @@
+{# Build template macros #}
+
+{% macro build_stats(container) %}
+    {% if container.builds %}
+        {% set invalid_builds =
+                container.builds | selectattr("valid", "false") | list %}
+        {% set valid_builds =
+                container.builds | selectattr("valid", "true") | list %}
+        {% set invalid_build_count = invalid_builds | length %}
+        {% set valid_build_count = valid_builds | length %}
+        {{- valid_build_count | string + " passed, " +
+            invalid_build_count | string + " failed" }}
+    {% endif %}
+{% endmacro %}
+
+{% macro container_summary(container, max_list_len) %}
+    {% if container.builds %}
+        {{- "\nBUILDS" }}
+        {% set invalid_builds =
+                container.builds | selectattr("valid", "false") | list %}
+        {% set invalid_build_count = invalid_builds | length %}
+        {% if invalid_builds %}
+            {{- "\n    Failures" }}
+            {% for origin, builds in invalid_builds|groupby("origin") %}
+                {% for build in builds %}
+                    {{- "       -"+  build.architecture + " (" + build.config_name + ")" }}
+                    {{- "\n       Build detail: https://kcidb.kernelci.org/d/build/build?orgId=1&var-id=" + build.id}}
+                    {% if build.log_excerpt %}
+                        {{- "       Build error: " + build.log_error }}
+                    {% endif %}
+                {% endfor %}
+                {{- "       CI system: " + origin + "\n\n"-}}
```

Comment:
I think spelling out which CI system reported which build is not really necessary, and makes the list harder to read. I think it's enough to list the contributing systems at the end of all builds, on one line.

Interspersing errors with CI system names makes it harder to read, and makes the list longer.

Ideally, it would be good to group builds by a particular error, and show how many builds had it. We can introduce that once we are able to encode errors into "issues", and be able to link to a particular issue for a particular revision.

Reply:
I won't hold the PR over this, though, if you prefer it this way. In the end it's up to recipients, how they want their notifications.

Reply:
Isn't it useful to report which CI system reported which failure?
I think it's good to have that info for further communication for a specific failure.
The same goes for test failures.
> Ideally, it would be good to group builds by a particular error, and show how many builds had it. We can introduce that once we are able to encode errors into "issues", and be able to link to a particular issue for a particular revision.

I have grouped failures based on architecture atm. We can change it later as required.

Reply:
> Isn't it useful to report which CI system reported which failure?

I don't think that's really important. More important is what the configuration, the command-line arguments, and the toolchain were. The CI system shouldn't matter, ultimately. And you can see it when you open the specific build URL. Certainly not something you need to know immediately.

But sure, please go ahead with what you've already built.